compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/BaseComposeScene.skiko.kt

```diff
@@ -104,11 +106,11 @@ internal abstract class BaseComposeScene(
 
     @Volatile
     private var hasPendingDraws = true
-    protected fun updateInvalidations() {
+    protected fun updateInvalidations(throttledToVsync: Boolean = false) {
```

Comment:
```suggestion
    protected fun updateInvalidations(throttledToVsync: Boolean) {
```
Better to  have explicit `throttledToVsync` to avoid missed fixes in another places

Reply:
The only case where `throttledToVsync` is `true` is for animations. Is it worth making everyone specify this explicitly?

Reply:
In my experience - yes, it is worth it.  Because:
- we made a significant deviation in behaviour
- future maintainers should think about it
- we can sometimes miss something (manually looking for usages is error prone)